src/libcommon/utility/utility.cpp

```diff
@@ -794,17 +791,24 @@ bool CommonUtility::fileNameIsValid(const SyncName &name) {
 }
 
 std::string CommonUtility::envVarValue(const std::string &name) {
+    bool isSet;
```

Comment:
Always initialize variables, even if in the present case it is not necessary. You never know how the method will evolve.

Reply:
Here is the commit with the initialization: [3898532](https://github.com/Infomaniak/desktop-kDrive/pull/274/commits/389853241d55592f8d984245d83903582256a484)